com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/message/Messages.java

```diff
@@ -30,6 +30,7 @@
     public static final String COULD_NOT_CREATE_VALID_HOST = "Could not create a valid host from \"{0}\"";
     public static final String COULD_NOT_CREATE_VALID_PORT = "Could not create a valid port from \"{0}\"";
     public static final String COULD_NOT_CREATE_VALID_ROUTE = "Could not create a valid route from \"{0}\"";
+    public static final String COULD_NOT_PARSE_ROUTE = "Cannot parse 'routes' property - check documentation for correct format.";

```

Comment:
This message is not consistent with the other ones from this group. Try to change it to Could not create valid routes from: ... or something similar.